tests/relay/test_rln.py

```diff
@@ -0,0 +1,19 @@
+import os
+import pytest
+from src.libs.custom_logger import get_custom_logger
+from src.steps.relay import StepsRelay
+
+logger = get_custom_logger(__name__)
+
+
+@pytest.mark.usefixtures("register_rln_main_relay_nodes")
+class TestRelayRLN(StepsRelay):
+    def test_register_rln(self):
+        logger.debug(f"Running register RLN test for main relay nodes")
+        key_stores_found = 0
+        for k in range(1, 3):
+            keystore_path = "/keystore_{k}/keystore.json"
```

Comment:
aren't you missing a f at the beginning of the string?
Ex: f"/keystore_{k}/keystore.json"

Reply:
Fixed at https://github.com/waku-org/waku-interop-tests/pull/16/commits/1421532e02d9bcea8bdf19ad3a54192e6b1a85c8